### PR TITLE
fix: mqtt topic extracting no longer requires all three fields

### LINKED ISSUE #10207: [MQTT topic parsing] Line Protocol elements should each be optional
### Relevent telegraf.conf

```toml
[[inputs.mqtt_consumer]]
    servers = ["tcp://localhost:1883"]
    topics = ["sensors/#"]
    data_format = "value"
    data_type = "string"
    [[inputs.mqtt_consumer.topic_parsing]]
        topic = "sensors/+"
        measurement = "measurement/_/"
        tags = "_/sensor_id"
```

### System info

Telegraf 1.21.0 rc0

### Docker

_No response_

### Steps to reproduce

See config above

Consider this case:
topic: `sensors/0011`
message: `45.0`

I want Line Protocol like: `sensors,sensor_id=0011 value=45.0`

Pub to topic: `sensors/0011` with message: `45.0`

I would configure: `measurement = "measurement/_"` and `tags = "_/sensor_id"` and leave fields alone.

### Expected behavior

Payloads may contain the necessary missing information for valid Line Protocol so there would be no need to address it in the topic.  Or field information is just not required if the user is okay with a `value=<value>` type of field.

### Actual behavior

The topic parsing configuration currently requires that `measurement`, `tags`, and `fields` params all be set and all underscores (i.e. `_/_/_`) does not circumvent this.  A common use case for topic parsing is going to be to add the parsed information to an existing payload.

### Additional info

_No response_

### PR DESCRIPTION
**Pull request by MyaLongmire**
### Required for all PRs:

- [ ] Updated associated README.md.
- [x] Wrote appropriate unit tests.
- [x] Pull request title or commits are in [conventional commit format](https://www.conventionalcommits.org/en/v1.0.0/#summary)

resolves #10207 

Extracting tags used to require all three fields but people may not use all three fields. Now checks if it is not there and does not error. Wrote a test with the config not having fields

### CODE REVIEW

**Comment by samhld** [comment]
Tested -- LGTM

File: plugins/inputs/mqtt_consumer/mqtt_consumer.go [hunk 1]
```diff
@@ -96,10 +96,6 @@ var sampleConfig = `
     "telegraf/+/mem",
     "sensors/#",
   ]
-  ## Enable extracting tag values from MQTT topics
-  ## _ denotes an ignored entry in the topic path
-  # topic_tags = "_/format/client/_"
-  # topic_measurement = "measurement/_/_/_"
```
Comment (sspaink) [inline plugins/inputs/mqtt_consumer/mqtt_consumer.go:102]: Did you mean to delete this?

Comment (MyaLongmire) [reply plugins/inputs/mqtt_consumer/mqtt_consumer.go:102]: Yes as it is further down in the readme and should not be in both places

Comment (sspaink) [reply plugins/inputs/mqtt_consumer/mqtt_consumer.go:102]: ah, good catch!

**Review by sspaink — APPROVED** [review]

**Pull request merged**